matkono/HealthSupApp
# Feature/patient flow

## What this PR does

- New look for the registration screen;
- Validate if text field is empty;
- Integrate with via_cep API;
- Autofill CEP information when CEP is valid;
- Check if CEP has invalid character or doesn't exist;

## How to check if this PR is correct

- Click in the icon ' + ' in the Patient homepage;
- Try to register a patient without filling the fields;
- Enter a valid CEP and you'll see that the fields below will appear;
- Enter a CEP code with invalid character like letters or a CEP that doesn't exist and you'll see an error message;

## Evidences

- New look to the registration screen:
![image](https://user-images.githubusercontent.com/44846533/107369608-c23d6280-6ac0-11eb-8f10-8b785f559821.png)

- Validate text field:
![image](https://user-images.githubusercontent.com/44846533/107370080-498ad600-6ac1-11eb-9fed-936604d76f0d.png)

- Autofill fields when CEP is valid:
![image](https://user-images.githubusercontent.com/44846533/107369789-f6188800-6ac0-11eb-9c02-ad9638b471e8.png)

- Error when CEP is invalid:
![image](https://user-images.githubusercontent.com/44846533/107370147-5c050f80-6ac1-11eb-80d1-70760b6be389.png)
